typescript/formsApps.d.ts

```diff
@@ -79,6 +79,7 @@ export type FormsAppHostnameConfiguration = {
     distributionDomain: string
   }
   errorMessage: null | string
+  recaptchaIntegrationPublicKey?: string
```

Comment:
I think we make this required and always set it to either the domain that have set up integrations or our public key.